### PR TITLE
Add RPC API reference - Closes #948

### LINKED ISSUE #948: Add new reference page for the WebSocket JSON 2.0 RPC API
See https://github.com/LiskHQ/lisk-service/blob/v0.2.0-rc.0/docs/api/websocket_json_rpc_api.md

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #948

### CODE REVIEW

File: modules/ROOT/pages/references/rpc-api.adoc [hunk 1]
```diff
@@ -0,0 +1,1101 @@
...
+----
+{
+    "method": "get.accounts",
+    "params": {
+        "address": "123L"
+    }
+}
+----
+
+==== get.votes
```
Comment (MichalTuleja) [inline modules/ROOT/pages/references/rpc-api.adoc]: `get.votes_sent`

Update: we've changed it to make it less ambiguous, the old API call `get.votes` still work as legacy API.

File: modules/ROOT/pages/references/rpc-api.adoc [hunk 2]
```diff
@@ -0,0 +1,1101 @@
...
+[source,json]
+----
+{
+    "method": "get.votes",
+    "params": {
+        "address": "4935562234363081651L"
+    }
+}
+----
+==== get.voters
```
Comment (MichalTuleja) [inline modules/ROOT/pages/references/rpc-api.adoc]: `get.votes_received`

File: modules/ROOT/pages/references/rpc-api.adoc [hunk 3]
```diff
@@ -0,0 +1,1101 @@
...
+  ],
+  "meta": {
+    "count": 100,
+    "offset": 25,
+    "total": 43749
+  },
+}
+----
+
+.Example: Get 20 item, skip 50 first
```
Comment (MichalTuleja) [inline modules/ROOT/pages/references/rpc-api.adoc]: "Get 20 items, ..."

File: modules/ROOT/pages/references/rpc-api.adoc [hunk 4]
```diff
@@ -0,0 +1,1101 @@
...
+}
+----
+.Example
+[source,json]
+----
+{
+    "method": "get.network.status"
+}
+----
+==== get.network.statistic
```
Comment (MichalTuleja) [inline modules/ROOT/pages/references/rpc-api.adoc]: `get.network.statistics`

**Review by MichalTuleja — CHANGES_REQUESTED** [review]
Looks good, a few minor issues found

File: modules/ROOT/pages/references/rpc-api.adoc [hunk 5]
```diff
@@ -0,0 +1,1101 @@
+= WebSocket JSON 2.0 RPC API
+:toc:
+
+== Lisk Service WebSocket JSON-RPC API Documentation
+
+=== Access paths and compatibility
+The WebSocket API can be accessed by the `wss://service.lisk.io/rpc-v1`.
+
+You can also access the testnet network by `wss://testnet-service.lisk.io/rpc-v1`.
```
Comment (chris529) [inline modules/ROOT/pages/references/rpc-api.adoc]:
```suggestion
The testnet network can also be accessed by `wss://testnet-service.lisk.io/rpc-v1`.
```

File: modules/ROOT/pages/references/rpc-api.adoc [hunk 6]
```diff
@@ -0,0 +1,1101 @@
...
+
+The Lisk Service WebSocket API uses the `socket.io` library and it is compatible with JSON-RPC 2.0 standard.
+The specification below contains numerous examples how to use the API in practice.
+
+=== Endpoint logic
+The logic of the endpoints comes as follows: the method naming is always based on the following pattern: `<action>.<entity>`, where the action is equivalent to HTTP standard (GET, POST, PUT, etc.) and `entity` is a part of the application logic, ex. `accounts`, transactions and so on.
```
Comment (chris529) [inline modules/ROOT/pages/references/rpc-api.adoc]:
```suggestion
The logic of the endpoints is derived as follows: the method naming is always based on the following pattern: `<action>.<entity>`, where the action is equivalent to HTTP standard (GET, POST, PUT, etc.) and `entity` is a part of the application logic, ex. `accounts`, transactions and so on.
```

File: modules/ROOT/pages/references/rpc-api.adoc [hunk 7]
```diff
@@ -0,0 +1,1101 @@
...
+{
+    "method": "get.transactions.statistics.month",
+    "params": {
+        "limit": 12,
+    }
+}
+----
+=== Network
+==== get.network.status
+Retrieves network details and constants such as network height, broadhash, fees, reward amount etc.
```
Comment (chris529) [inline modules/ROOT/pages/references/rpc-api.adoc]:
```suggestion
Retrieves network details and constants such as network height, broadhash, fees, reward amount, etc.
```

File: modules/ROOT/pages/references/rpc-api.adoc [hunk 8]
```diff
@@ -0,0 +1,1101 @@
...
+----
+.Example
+[source,json]
+----
+{
+    "method": "get.network.status"
+}
+----
+==== get.network.statistics
+Retrieves network statistics such as number of peers, node versions, heights etc.
```
Comment (chris529) [inline modules/ROOT/pages/references/rpc-api.adoc]:
```suggestion
Retrieves network statistics such as the number of peers, node versions, heights, etc.
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a couple of suggestions, and a couple of minor points 👍

File: modules/ROOT/pages/references/rpc-api.adoc [hunk 9]
```diff
@@ -0,0 +1,1101 @@
...
+    "meta": {
+      "count": 100,
+      "offset": 25,
+      "total": 43749
+    }
+  }
+]
+----
+
+.Example: Get address with certain Lisk account ID
```
Comment (chris529) [inline modules/ROOT/pages/references/rpc-api.adoc]:
```suggestion
.Example: Get address with a specific Lisk account ID
```

File: modules/ROOT/pages/references/rpc-api.adoc [hunk 10]
```diff
@@ -0,0 +1,1101 @@
...
+    }
+  ],
+  "meta": {
+    "count": 100,
+    "offset": 25,
+    "total": 43749
+  }
+}
+----
+.Example: Get hosts with certain IP
```
Comment (chris529) [inline modules/ROOT/pages/references/rpc-api.adoc]:
```suggestion
.Example: Get hosts with a specific IP address
```

**Review by chris529 — COMMENTED** [review]
Looks great to me Mona, although I just noticed 2 headlines that I think would appear to be clearer to the reader with my suggestions. Let me know your thoughts 👍

File: modules/ROOT/pages/references/rpc-api.adoc (hunk 9 above)
Comment (Tschakki) [reply modules/ROOT/pages/references/rpc-api.adoc]:
```suggestion
.Example: Get account with a specific Lisk account ID
```

**Review by MichalTuleja — APPROVED** [review]

**Pull request merged**